024-DoYouEvenSynapse/Coach/00-prereqs.md

```diff
@@ -0,0 +1,14 @@
+# Challenge 0: Pre-requisites - Ready, Set, GO! 
+
+**[Home](README.md)** - [Next Challenge >](./01-data-gathering.md)
+
+## Notes and Guidance
+- Make sure that you have joined the Teams group for this track. The first person/coach on your team at your table should create a new channel in this Team with your team name.
+- Install the recommended toolset:
+    - Web browser to interact with Azure Portal and web-based data tools
+    - [Power BI Desktop](https://powerbi.microsoft.com/en-us/desktop/)
+- Make sure your team creates the Teams channel for their team
+- PowerBI Desktop is to be installed on their desktops
+- Access to a web browser should be allowed as well
+- Run the lab notebook and make the output data ("/dailyReports/", "/initialData/") available
```

Comment:
I assume this refers to the Jupiter notebook in the /Coach/Resources folder.   

You should probably make the instructions for a potential coach who is not a Jupiter guru clearer.   (For example, I have no idea what to do with this instruction! ;))

Reply:
Also please wrap the file paths, ie: /dailyReports/ in backticks instead of quotes so they appear monospaced.

Reply:
Amended with Orrin